lib/tweetstream/version.rb

```diff
@@ -1,3 +1,3 @@
 module TweetStream
-  VERSION = '2.6.1'
+  VERSION = '2.6.2'
```

Comment:
please revert this change, we prefer to change this only when cutting releases, thanks